src/test/java/net/liquidpineapple/pang/LevelEditorTest.java

```diff
@@ -1,52 +1,64 @@
 package net.liquidpineapple.pang;
 
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
-import static org.mockito.Matchers.any;
 import static org.mockito.internal.verification.VerificationModeFactory.times;
 
 import net.liquidpineapple.pang.gui.Board;
+import net.liquidpineapple.pang.gui.LifeSystem;
 import net.liquidpineapple.pang.objects.Ball;
 import net.liquidpineapple.pang.objects.BallMovement;
+import net.liquidpineapple.pang.objects.GameObject;
 import net.liquidpineapple.pang.objects.Player;
+import net.liquidpineapple.pang.screens.ControlsScreen;
 import net.liquidpineapple.pang.screens.LevelEditor;
+import net.liquidpineapple.pang.screens.Screen;
+import net.liquidpineapple.pang.screens.UserCreatedLevels;
+import net.liquidpineapple.pang.screens.WinScreen;
 
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import org.mockito.Mock;
 import org.mockito.Mockito;
 import org.powermock.api.mockito.PowerMockito;
 import org.powermock.core.classloader.annotations.PrepareForTest;
 import org.powermock.modules.junit4.PowerMockRunner;
 import org.powermock.reflect.Whitebox;
 
 import java.awt.Component;
-import java.awt.MouseInfo;
 import java.awt.Point;
 import java.awt.PointerInfo;
+import java.awt.Robot;
 import java.awt.event.KeyEvent;
+import java.awt.event.MouseEvent;
+import java.util.ArrayList;
 
 
 @RunWith(PowerMockRunner.class)
-@PrepareForTest({InputHandler.class, Application.class, Board.class} )
+@PrepareForTest({InputHandler.class, Application.class, Board.class})
 public class LevelEditorTest {
   private LevelEditor levelEditor;
   private PointerInfo pointerInfo;
   private InputHandler inputHandler;
   private Application app;
   private Board board;
+  private Robot robot;
 
   @Before
   public void setUp() throws Exception {
     levelEditor = PowerMockito.spy(new LevelEditor());
     inputHandler = new InputHandler();
     board = Mockito.mock(Board.class);
     pointerInfo = Mockito.mock(PointerInfo.class);
-    Mockito.when(pointerInfo.getLocation()).thenReturn(new Point(100,100));
-    Mockito.when(board.getLocationOnScreen()).thenReturn(new Point(0,0));
+    Mockito.when(pointerInfo.getLocation()).thenReturn(new Point(100, 100));
+    Mockito.when(board.getLocationOnScreen()).thenReturn(new Point(0, 0));
+    PowerMockito.mockStatic(Application.class);
+    Mockito.when(Application.getBoard()).thenReturn(board);
+    robot = new Robot();
+    robot.mouseMove(100, 100);
```

Comment:
Using a Robot to move the mouse is a bad idea if you dont use asserts. Its better to remove this.